src/leader_scheduler.rs

```diff
@@ -18,11 +18,11 @@ use solana_sdk::vote_transaction::VoteTransaction;
 use std::io::Cursor;
 use std::sync::Arc;
 
-pub const TICKS_PER_BLOCK: u64 = 4;
-pub const DEFAULT_BOOTSTRAP_HEIGHT: u64 = TICKS_PER_BLOCK * 256;
-pub const DEFAULT_LEADER_ROTATION_INTERVAL: u64 = TICKS_PER_BLOCK * 32;
-pub const DEFAULT_SEED_ROTATION_INTERVAL: u64 = TICKS_PER_BLOCK * 256;
-pub const DEFAULT_ACTIVE_WINDOW_LENGTH: u64 = TICKS_PER_BLOCK * 256;
+pub const DEFAULT_TICKS_PER_SLOT: u64 = 8;
+pub const DEFAULT_BOOTSTRAP_HEIGHT: u64 = 1024;
+pub const DEFAULT_SLOTS_PER_EPOCH: u64 = 8;
```

Comment:
let's have some comments about the implications and why these have been chosen (or at least why they're a default)

I see votes every 800ms, bootstrap of ~10 seconds, 6.4 second epochs, is that right?

Reply:
Any bootstrap time at all sounds like we're not acknowledging @mvines's efforts to move bootstrapping into the genesis block. Can we do a zero-time bootstrap?

Reply:
zero-time bootstrap blows up a bunch of tests.  I’ve got most of those fixed in my PR already.  Let’s get this sucker landed so I can rebase on it and we can move forward